### ISSUE TITLE
Consider bycatch table in assessment

### ISSUE THREAD

**Issue by aaronmberger-nwfsc** [pacific-hake/hake-assessment#962]
I saw this in the AK Pollock assessment and thought it was useful.  We can at least think about how easy/difficult it would be to create this on an annual basis.

![image](https://user-images.githubusercontent.com/7946234/199802160-f7c56c4a-b69f-4383-8647-aec7ec271a2b.png)

**Comment by cgrandin** [comment]
I've had a lot of people ask about Salmon bycatch in Canada so it would be good to just refer them to a table

**Comment by andrew-edwards** [comment]
It'd be good to have. Rowan did something similar in some of our rockfish assessments - I expect he has code. I can look into it at some point.

**Comment by kellijohnson-NOAA** [comment]
I worry about confidentiality 🤕

**Comment by aaronmberger-nwfsc** [comment]
Yes, we would need to be careful about that.  I was thinking something like the [PacFin report](https://reports.psmfc.org/pacfin/f?p=501:202::INITIAL:NO:::).  We can discuss next Monday.

**Comment by andrew-edwards** [comment]
Here's what we've done in the past, this is POP off West Coast Vancouver Island (I've asked someone to point me to the code):
![image](https://user-images.githubusercontent.com/6977190/200379690-d427de02-23cc-4aa4-967e-65a8f43f2b50.png)

**Comment by andrew-edwards** [comment]
Nice to have, but no-one's really asked for them, so low priority. 

Plus adds more time to web accessibility tagging.

**Comment by andrew-edwards** [comment]
Just came across this (for reference, or if anyone asks)...  The MSC Surveillance report has a table for each of Canada and US, see page 16 of https://fisheries.msc.org/en/fisheries/pacific-hake-mid-water-trawl/@@assessment-documentsets?assessment_step=Surveillance+Audit&documentset_name=Surveillance+report&assessment_id=FA-02546&phase_name=Ongoing+surveillance&start_date=2020-11-03  

Not exactly sure where the numbers come from (might have to dig in, but presumably NOAA and DFO), but we could always just quote a few to get something in the document without adding extra tables.

**Comment by aaronmberger-nwfsc** [comment]
I'm adding a 2024 milestone on this if we find time to do it for next year

**Comment by kellijohnson-NOAA** [comment]
@aaronmberger-nwfsc can we punt this to 2025?
